arex-instrumentation/common/arex-reactorcore/src/main/java/io/arex/inst/reactorcore/common/MonoToFutureWrapper.java

```diff
@@ -0,0 +1,15 @@
+package io.arex.inst.reactorcore.common;
+
+import io.arex.agent.bootstrap.ctx.TraceTransmitter;
+import java.util.concurrent.CompletableFuture;
+
+public class MonoToFutureWrapper {
+
+    public static<T> CompletableFuture<T> thenApplyAsync(CompletableFuture<T> future,TraceTransmitter tm){
+        return future.thenApplyAsync(content->{
```

Comment:
thenRunAsync cannot guarantee the order of execution, does this meet your requirements?